.circleci/config.yml

```diff
@@ -30,7 +32,10 @@ commands:
       - restore_cache:
           keys: # Use gradle version for cache, so it doesn't download every time
             - v1-dependencies-{{ checksum "./gradle/wrapper/gradle-wrapper.properties" }}
-      - setup_remote_docker
+      # Update this according to the output of the image used to override the default of Docker 17
+      #   Ex. docker run -it --rm cimg/openjdk:14.0.2 docker -v
+      - setup_remote_docker:
+          version: 19.03.11
```

Comment:
`19.03.12` is latest available https://circleci.com/docs/2.0/building-docker-images/#docker-version

Reply:
that version is on the cimg/base image, not the java one, hence the instructions mentioning check the image that we are using. I think most are using cimg/base to run docker commands, but we can't as we invoke docker with gradle.

Reply:
the `cimg/base:stable` got me there (because of the java requirement), but the change looks good.